extra_data/export.py

```diff
@@ -225,6 +234,9 @@ def serve_files(path, port, source_glob='*', key_glob='*', **kwargs):
         Path to the HDF5 file or file folder.
     port: int
         Local TCP port to bind socket to.
+    data_like: str
```

Comment:
this should be also changed if you change the cli argument name

Reply:
Done